src/HexViewer.ts

```diff
@@ -100,11 +100,10 @@ export class HexViewer {
     private content: JQuery;
     private contentOuter: JQuery;
 
-    public mouseDownOffset: number;
     private canDeselect: boolean;
-
     public selectionStart: number = -1;
     public selectionEnd: number = -1;
+    public selectionCursor: number = -1;
```

Comment:
Again, I think that the special variable for `selectionCursor` is unnecessary, because always applies `selectionCursor == selectionStart` or `selectionCursor == selectionEnd`, nothing between or outside. Why don't introduce a single property `isSelectionCursorAtEnd` which just distinguishes between these two states?